charts/aws-efs-csi-driver/values.yaml

```diff
@@ -110,6 +115,11 @@ node:
     #   cpu: 100m
     #   memory: 128Mi
   nodeSelector: {}
+  updateStrategy: 
+   type: RollingUpdate
```

Comment:
We should not change the strategy for node deployment but only add the possibility to change it since this PR says exactly that. So this should be `updateStrategy: {}` and the rest commented out.

Reply:
done